iq_user_base.module

```diff
@@ -54,10 +54,15 @@ function iq_user_base_theme($existing, $type, $theme, $path) {
  * Implements hook_preprocess_block__userareamenu().
  */
 function iq_user_base_preprocess_block__userareamenu(&$variables) {
+  // Get username.
   if (!in_array('anonymous', $variables['user']->getRoles())) {
     $variables['name'] = $variables['user']->getDisplayName();
   }
   $variables['destination'] = Url::fromRoute('<current>')->toString();
+
+  // Get current language.
+  $language = \Drupal::languageManager()->getCurrentLanguage()->getId();
```

Comment:
Please use getCurrentLanguage(\Drupal\Core\Language\LanguageInterface::TYPE_CONTENT) to make sure, we always use content language.